utils/MonitoredResource.md

```diff
@@ -1,84 +0,0 @@
-# Monitored Resource
```

Comment:
I think this page can be left as additional information for the "Auto-detection" section.

Reply:
My thinking was that it's more scalable to keep these docs close to the code they belong to. OTOH, with spread across languages, this repo may indeed still be the better option – maybe with separate file per detection mechanism then as those will probably grow.